### PR TITLE
[ADD] website_hr_department

### PR DESCRIPTION
**Pull request by lmignon**
New module to display the departments in the website

### CODE REVIEW

File: website_hr_department/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+#     You should have received a copy of the
+#     GNU Affero General Public License
+#     along with website_hr_department.
+#     If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': "Departments Page",
+    'summary': """
+        Display the structure of your departments and their members.
```
Comment (pedrobaeza) [inline website_hr_department/__openerp__.py:27]: Don't put triple quota on summary. It introduces line breaks.

**Comment by sbidoul** [comment]
:+1: (functional test only)

File: website_hr_department/__openerp__.py (hunk 1 above)
Comment (sbidoul) [inline website_hr_department/__openerp__.py:27]: Note this style comes from the Odoo default module template:
https://github.com/odoo/odoo/blob/8.0/openerp/cli/templates/default/__openerp__.py

Comment (pedrobaeza) [inline website_hr_department/__openerp__.py:27]: Well, have you tested if it appears well?

Comment (lmignon) [inline website_hr_department/__openerp__.py:27]: yes it appears well
image
image

Comment (pedrobaeza) [inline website_hr_department/__openerp__.py:27]: OK then, thanks for the explanation

**Comment by pedrobaeza** [comment]
:+1: (code review)

**Comment by hbrunn** [comment]
:+1:

**Pull request merged**